src/main/resources/org/jenkinsci/plugins/workflow/libs/FolderLibraries/config.jelly

```diff
@@ -29,14 +29,18 @@ THE SOFTWARE.
         <f:block>
             <j:out value="${%blurb}"/>
         </f:block>
-        <f:entry>
-            <f:repeatableProperty field="libraries" header="${%Library}">
-                <f:entry title="">
-                    <div align="right">
-                        <f:repeatableDeleteButton/>
-                    </div>
+        <f:block>
```

Comment:
[for reviewers](https://github.com/jenkinsci/workflow-cps-global-lib-plugin/pull/39/files?w=1)